### ISSUE TITLE
Resource is prefixed with '/' 

### ISSUE THREAD

**Issue by amangale** [OptimalBits/node_acl#124]
Hi,

I have a resource set up like acl.allow('admin', 'accounts', ['*']).

The debug info I am getting from using acl.middleware() is:

Not allowed get on /accounts by user 1. 

Now, the user id is correct but why is the resource '/accounts' instead of 'accounts'.

thanks.

**Comment by jonmassot** [comment]
`/accounts` might be referring to the route (path) off the application, probably coming from express.Router. You can mod the `acl.allow('admin', '/accounts', ['*'])` if you don't want to manage the different resources to fit node_acl. Note that using `/` might cause other problems (ie: mongo does not support `/`).

**Comment by amangale** [comment]
Yes. I am using redis backend. The only way, I can get the acl.midleware() to work is if the '/' is used.

So, the acl.allow block essentially list the routes as resources. Is this correct or is there other way to use the acl.middleware() for express.

thanks.

**Comment by manast** [comment]
@amangale the middleware for express is quite simple, you can easily write one that satisfies your requirements:
https://github.com/OptimalBits/node_acl/blob/master/lib/acl.js#L545-L616
